src/xdocs/contributing.xml

```diff
@@ -67,7 +67,7 @@
 
       <p>
         To report issue please follow our best practices page - <a href="report_issue.html">
-          How to report an issue ?</a> (<a href="report_issue.html#How_to_report_a_bug">
+          How to report an issue ?</a> (<a href="report_issue.html#How_to_report_a_bug.3F">
```

Comment:
May be remove whitespace before question mark as well?
Same in line 71

Reply:
done